Aptoide/aptoide-client-v8
# [AS-94] Created EditorialList fragment

**What does this PR do?**

 Creates a view that shows all active curation cards

**Database changed?**

 No

**Where should the reviewer start?**

EditorialListFragment.java

**How should this be manually tested?**

 Go to the editorial BN view and see if it navigates to the correct editorial, shows the correct information and if it breaks

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/AS-94 

**Code Review Checklist**
